webapp/src/Controller/API/AbstractRestController.php

```diff
@@ -137,11 +137,15 @@ protected function renderCreateData(
         $params = [
             'id' => $id,
         ];
+        $postfix = '';
         if ($routeType !== 'user') {
             $params['cid'] = $request->attributes->get('cid');
+            if ($params['cid'] === null) {
+                $postfix = '_1';
+            }
         }
         $headers = [
-            'Location' => $this->generateUrl("v4_app_api_{$routeType}_single", $params, UrlGeneratorInterface::ABSOLUTE_URL),
+            'Location' => $this->generateUrl("v4_app_api_{$routeType}_single$postfix", $params, UrlGeneratorInterface::ABSOLUTE_URL),
```

Comment:
```suggestion
            'Location' => $this->generateUrl("v4_app_api_{$routeType}_single" . $postfix, $params, UrlGeneratorInterface::ABSOLUTE_URL),
```
I find it nicer to read, assuming it does work.

Reply:
`"v4_app_api_{$routeType}_single{$postfix}"` would also be fine for me, I prefer to directly see that we do something with a variable there (in this case twice).